### PR TITLE
Pull request for current sprint ending March 17/18th

### PR DESCRIPTION
**Pull request by jmccausland**
@garys-esri please review pull request for version 0.5.0 for public repo

### CODE REVIEW

**Comment by garys-esri** [comment]
Created issue #53 to resolve after this pull request

**Comment by garys-esri** [comment]
@jmccausland I can't figure out how to edit a polyline message. I can select it so it turns pink, but I don't see vertices. How do I do it?

File: source/MilitaryPlanner/Views/MapView.xaml [hunk 1]
```diff
@@ -50,8 +50,10 @@
             </i:Interaction.Behaviors>
             <esri:Map x:Name="MyMap">
                 <!-- ArcGIS Online Tiled Basemap Layer -->
-                <esri:ArcGISTiledMapServiceLayer ID="World Topo Map" ServiceUri="http://services.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer"/>
-                <esri:ArcGISTiledMapServiceLayer ID="TestMapServiceLayer" ServiceUri="http://sampleserver6.arcgisonline.com/arcgis/rest/services/Toronto/ImageServer" />
```
Comment (garys-esri) [inline source/MilitaryPlanner/Views/MapView.xaml]: MapViewModel.cs depends on having a layer called TestMapServiceLayer. I think this causes #54.

**Comment by garys-esri** [comment]
#54 should be resolved before merging.

**Comment by jmccausland** [comment]
@garys-esri currently there is no UI to invoke the editor, for now use the 'F2' function key to start an edit on a selected polyline or polygon, press 'F2' again to save or 'ESC' to cancel.  Also 'CTRL-Z' and 'CTRL-Y' will do the usual undo and redo commands while editing.

**Pull request merged**